lib/sqlalchemy/orm/decl_api.py

```diff
@@ -809,6 +809,8 @@ class Base(metaclass=DeclarativeMeta):
         class_dict["__abstract__"] = True
         if mapper:
             class_dict["__mapper_cls__"] = mapper
+        if cls.__class_getitem__:
```

Comment:
so we'd like this to port to SQLAlchemy 1.4, which still runs on Python 2, so you likely need to not assume this attribute is present.

Reply:
done